### PR TITLE
zstd

### PR DESCRIPTION
**Pull request by devsnek**
adds zstd-stream support

### CODE REVIEW

File: src/WebSocket.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ exports.pack = erlpack ? erlpack.pack : JSON.stringify;
 
 exports.unpack = data => {
   if (!erlpack || data[0] === '{') return JSON.parse(data);
-  if (!(data instanceof Buffer)) data = Buffer.from(new Uint8Array(data));
+  if (!(data instanceof Buffer)) data = Buffer.from(data);
```
Comment (MrJacz) [inline src/WebSocket.js]: Shouldn't this be `Buffer.isBuffer(data)`?

**Comment by appellation** [comment]
Can you at least attempt to give a meaningful description, even if you've already deleted the PR template?

File: README.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ For production bots, using node-opus should be considered a necessity, especiall
 
 ### Optional packages
 - [zlib-sync](https://www.npmjs.com/package/zlib-sync) for significantly faster WebSocket data inflation (`npm install zlib-sync`)
+- [zucc](https://www.npmjs.com/package/zucc) for even more significantly faster WebSocket data inflation (`npm install zucc`)
```
Comment (amishshah) [inline README.md]: please fix this

File: README.md [hunk 3]
```diff
@@ -41,6 +41,7 @@ For production bots, using node-opus should be considered a necessity, especiall
 
 ### Optional packages
 - [zlib-sync](https://www.npmjs.com/package/zlib-sync) for significantly faster WebSocket data inflation (`npm install zlib-sync`)
+- [zucc](https://www.npmjs.com/package/zucc) for MAXIMUM WebSocket data inflation performance (`npm install zucc`)
```
Comment (amishshah) [inline README.md]: can you prove it is a maximum?

**Pull request merged**